presto-docs/README.md

```diff
@@ -156,4 +156,7 @@ Maven pom has already moved to the next SNAPSHOT version.
 - Different installation methods for Sphinx result in different versions, and
   hence in sometimes different problems. Especially when also using
   sphinx-autobuild we recommend using the `pip`-based installation.
+- Usage of sphinx 2.x and higher fails due to parallel write being not supported
+  by the sitemap extension. As a workaround clear SPHINXOPTS in your command
+  line call of `make SPHINXOPTS="" clean html` or in the `Makefile` itself
```

Comment:
Some thoughts

* We should recommend using an older version, since there may be other compatibility issues.
* We shouldn't recommend changing the Makefile, as that will cause problems if people accidentally commit it (or they'll have to go through effort to keep it as a local change).

In general, we should tell people the recommend approach, rather than giving them every possible option. If they know what they're doing and want to deviate, that's fine, otherwise they should follow the recommendation.

```markdown
- Sphinx 2.x+ fails due to requiring parallel write support, which our sitemap extension
  does not support. We recommend installing an older version by running
  `pip3 install sphinx==1.8.2`. Alternatively, you clear `SPHINXOPTS` when running
  Sphinx by using `make SPHINXOPTS="" clean html`, but this may result in other
  compatibility issues or differences from the output produced by the Maven plugin.

Reply:
Sure .... we are telling normal users to use Maven .... all the sphinx installation tips are left out on purpose. At this stage I dont want to write any tips for installing sphinx which is why I left all that out and just added it as a known issue ... if you read those you are automatically an advanced user because you scrolled down ..